### ISSUE TITLE
Use CLI project functionality in Kitty Items

### ISSUE THREAD

**Issue by psiemens** [onflow/kitty-items#19]
Once the new `flow project` command is merged and released, we should update this repository to make use of the new `flow.json` structure, including the automated contract deployment feature.

Here's what that should include:

- [x] `flow.json` file committed to this repository
- [x] `flow.json` file includes paths to contracts in `kitty-items-cadence`
- [x] `flow.json` file includes configuration for the Flow Emulator
- [x] `flow project deploy` works with the configured emulator
- [x] `kitty-items-deployer` is removed (no longer needed)
